proto/one/operations/sample/analyte_results_import_telemetry.proto

```diff
@@ -6,21 +6,16 @@ option csharp_namespace ="ONE.Models.CSharp";
 
 message AnalyteResultsImportTelemetry {
 
-    // Name of the result.
+    // Name of the telemetry. Ex. SampleNumber, Analyte, Result...
     string name = 1;
 
-    // The column number that contains the value, zero based.
+    // The column number that contains the value of the telemetry, zero based.
     int32 valueColumnNumber = 2;
 
-    // Optional properties below, only used for narrow format csv files.
-    // These files usually contain 1 record per file.
+    // Optional properties below.
 
-    // The filter name of the result to search for in the file.
-    // Needed because this may differ from name above.
-    google.protobuf.StringValue filter = 3;
-
-    // The column number that contains the filter name.
-    google.protobuf.Int32Value filterByColumnNumber = 4;
+    // The name of the telemetry (only used with wide format).
```

Comment:
Seems like you have 'Name of the telemetry' in here twice?

Reply:
Looks weird but they serve different purposes which are documented here if you're interested: https://aquaticinformatics.atlassian.net/wiki/spaces/OP/pages/422183066/Schema+Configuration+Data+Analysis+Results+Import+Profile